example/lib/main.dart

```diff
@@ -12,20 +15,14 @@ class MyApp extends StatefulWidget {
   _MyAppState createState() => _MyAppState();
 }
 
-enum OpState {
-  notStarted,
-  inProgress,
-  succeeded,
-  failed,
-}
-
-const defaultChannel = 'test-channel';
+const defaultChannel = Constants.channelName;
 
 class _MyAppState extends State<MyApp> {
   String _platformVersion = 'Unknown';
   String _ablyVersion = 'Unknown';
-  OpState _provisioningState = OpState.notStarted;
-  provisioning.AppKey? _appKey;
+
+  // final String _apiKey = const String.fromEnvironment(Constants.ablyApiKey);
+  String _apiKey = const String.fromEnvironment(Constants.ablyApiKey);
```

Comment:
I know we discussed this elsewhere but is it now documented in the readme?

Reply:
Yup, this is documented in the [README's Running the example
 section](https://github.com/ably/ably-flutter/tree/feature/push-notifications#running-the-example), for example: 
```
If starting the app using the command line, run `flutter run --dart-define=ABLY_API_KEY=put_your_api_key_here`
```

Reply:
I think with the update to `.run/example.run.xml` the `--dart-defined` is already included for Android studio run configuration. We can just mention to set the API key at end of `Additional run args`

Reply:
The instruction for that is also in the [section](https://github.com/ably/ably-flutter/tree/feature/push-notifications#running-the-example) 
> - If starting the app using Android Studio:
>   - Edit the example app run configuration add the --dart-define flag to additional run args

I took this opportunity to add more detail and improve the language in 6a4ca945e8ff5bc100abd7a7346337271c2f25b2

[Click for Preview](https://github.com/ably/ably-flutter/blob/5c128027bcb8ba91e4ec8b1df25d968c93334e57/README.md)